src/App.module.css

```diff
@@ -0,0 +1,26 @@
+.container {
+    display: grid;
+    height: 100%;
```

Comment:
I wonder if the 100% heights are necessary here and for the sidebar. Seems to be fine without.

Reply:
It's not needed for most pages, but the 'Analytics Tables' and 'Min-Max Value Generation' pages seem cutoff without it since the sidebar becomes too short.

| Without 100% height | With 100% height |
--- | ---
| ![image](https://user-images.githubusercontent.com/4295266/119517885-e96a2800-bd6f-11eb-9ead-95780eb35b9e.png) | ![image](https://user-images.githubusercontent.com/4295266/119518051-13bbe580-bd70-11eb-8bb9-a9476887b771.png) |

Reply:
Ah ok I didn't notice that 👍